config/application.rb

```diff
@@ -39,7 +39,7 @@ class Application < Rails::Application
     config.api_only = true
     config.middleware.insert_before 0, Rack::Cors do
       allow do
-        origins ENV['CORS_ORIGIN']
```

Comment:
If you copy `.env.development.local.template` into a new `.env.development.local` then `ENV['CORS_ORIGIN']` should have the right dev origin to allow, so we don't have to include this change

Reply:
(y)